ci-operator/jobs/openshift/jenkins/openshift-jenkins-master-presubmits.yaml

```diff
@@ -60,6 +60,40 @@ presubmits:
           - configMap:
               name: cluster-profile-gcp
     trigger: ((?m)^/test( all| e2e),?(\s+|$))
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/images
+    decorate: true
+    name: pull-ci-openshift-jenkins-master-images
```

Comment:
Not sure about this one, we may have to black list this out.  Jenkins images aren't part of our payload yet.

Reply:
Oops, I meant to put this on the post submit promotion.